oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/IndexImproperUsageCommonTest.java

```diff
@@ -148,12 +148,22 @@ public void queryFilterRegexRestrictionsWarn() throws Exception {
         root.commit();
 
         assertEventually(() -> {
-            assertThat(explain("select [jcr:path] from [nt:base] where [propa] = \"oak\""), containsString(indexOptions.getIndexType() + ":" + indexName));
-            // List appender should not have any warn logs as we are searching under right descendant as per path restrictions
+        	assertTrue(explain("select [jcr:path] from [nt:base] where [propa] = \"oak\"").contains(indexOptions.getIndexType() + ":" + indexName));
+            // List appender should not have any warn logs as we are searching using a term matching regex
             assertFalse(isWarnMessagePresent(listAppender ,String.format(QUERY_FILTER_WARN_MESSAGE, indexName, regex)));
```

Comment:
this is not right. With the changes to `QUERY_FILTER_WARN_MESSAGE`, now `String.format` expects 3 arguments. I think `"oak"` should be added.